tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -507,7 +501,18 @@ class CompletionArgSuite extends BaseCompletionSuite {
         |}
         |""".stripMargin,
     """|x: Int
+       |x = : Byte
```

Comment:
For the future, this actually shouldn't be allowed for infix arguments, could you create an issue?

Reply:
fixed it